template/package.json

```diff
@@ -39,5 +40,28 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "eslint": "^7.22.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-cypress": "^2.11.2",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.3.1",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "husky": "^4.3.8",
```

Comment:
How about bringing the husky [5+ version](https://github.com/typicode/husky/releases/tag/v5.0.0)? The configuration is significantly different for this new version, but I think it is better.

Reply:
I'm getting a strange error and I really don't know why it's happening. Maybe this will take longer than I thought.

Reply:
I think now I got it.